service-registry/src/main/java/io/servicecomb/serviceregistry/definition/DefinitionConst.java

```diff
@@ -25,13 +25,13 @@
 
   String nameKey = "name";
 
-  String qulifiedServiceNameKey = serviceDescriptionKey + "." + nameKey;
```

Comment:
It could be better if we use the up case letter to specify the constant string name. Such as QUALIFIED_SERVICE_NAME_KEY can let the user know it's a constant string.

Reply:
It seems constants in `DefinitionConst.java` are all in camel case. Besides, there are some duplicate constants in `DefinitionConst.java` and `ServiceCombConstants.java`. Maybe another PR for this as it may affect others and this PR is mainly focus on misspell issues?

Reply:
OK， you can fill a JIRA to check this issue.